test/badges.test.ts

```diff
@@ -200,188 +204,146 @@ describe('Badge Specs', () => {
     const { badgesContract, typedData } = await loadFixture(deployContractFixture)
     const offChainHash = _TypedDataEncoder.hash(typedData.domain, typedData.types, typedData.value)
-
     const onChainHash = await badgesContract.getHash(
       typedData.value.active,
       typedData.value.passive,
       typedData.value.tokenURI
     )
-
     expect(offChainHash).to.equal(onChainHash)
   })
-
   it('should successfully mint badge', async function () {
     // deploy contracts
     const { badgesContract, raftContract, typedData, issuer, claimant, owner } = await loadFixture(
       deployContractFixture
     )
     const specUri = typedData.value.tokenURI
-
     // mint raft
     const { raftTokenId } = await mintRaftToken(raftContract, issuer.address, specUri, owner)
-
     // create spec
     await createSpec(badgesContract, specUri, raftTokenId, issuer)
-
     // get signature
     const { compact } = await getSignature(typedData.domain, typedData.types, typedData.value, issuer)
-
     // take's "from" is the issuer
     // take's msg.sender is the claimant
     const txn = await badgesContract.connect(claimant).take(typedData.value.passive, typedData.value.tokenURI, compact)
     await txn.wait()
-
     const transferEventData = await getTransferEventLogData(txn.hash, badgesContract)
     // currently the ERC4973 is hardcoding from to 0 in the event - https://github.com/rugpullindex/ERC4973/issues/39
     // expect(transferEventData.from).equal(typedData.value.passive) // issuer.address,
     expect(transferEventData.to).equal(typedData.value.active) // claimant.address
     expect(transferEventData.tokenId).gt(0)
-
     const badgeEventData = await getBadgeMintedEventLogData(txn.hash, badgesContract)
     expect(badgeEventData.to).equal(typedData.value.active) // claimant.address
-
     expect(badgeEventData.specUri).equal(specUri)
-
     expect(badgeEventData.tokenId).gt(0)
-
     const ownerOfMintedToken = await badgesContract.ownerOf(transferEventData.tokenId)
     expect(ownerOfMintedToken).to.equal(claimant.address)
-
     const balanceOfClaimant = await badgesContract.balanceOf(claimant.address)
     expect(balanceOfClaimant).to.equal(1)
-
     const uriOfToken = await badgesContract.tokenURI(transferEventData.tokenId)
     expect(uriOfToken).to.equal(typedData.value.tokenURI)
   })
-
   it('should fail to mint badge when trying as an unauthorized claimant', async () => {
     // deploy contracts
     const { badgesContract, raftContract, typedData, issuer, claimant, owner } = await loadFixture(
       deployContractFixture
     )
     const specUri = typedData.value.tokenURI
-
     // mint raft
     const { raftTokenId } = await mintRaftToken(raftContract, issuer.address, specUri, owner)
-
     // create spec
     await createSpec(badgesContract, specUri, raftTokenId, issuer)
-
     // get signature
     const { compact } = await getSignature(typedData.domain, typedData.types, typedData.value, issuer)
-
```

Comment:
were these newlines auto removed or did u manually remove them? 
I personally find it hard to generally read code without separation of logical blocks.